Description: Don't list key without schema in gnome-settings-daemon.convert. It crashes gsettings-data-convert
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: no

--- a/data/gnome-settings-daemon.convert
+++ b/data/gnome-settings-daemon.convert
@@ -42,7 +42,6 @@ priority = /apps/gnome_settings_daemon/p
 [org.gnome.settings-daemon.plugins.keyboard]
 active = /apps/gnome_settings_daemon/plugins/keyboard/active
 priority = /apps/gnome_settings_daemon/plugins/keyboard/priority
-show-keyboard-leds-indicator = /desktop/gnome/peripherals/keyboard/general/duplicate_leds
 
 [org.gnome.settings-daemon.plugins.media-keys]
 active = /apps/gnome_settings_daemon/plugins/keybindings/active
